app/main.py

```diff
@@ -1,25 +1,77 @@
-from abc import ABC
+from abc import ABC, abstractmethod
+from typing import Type
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Type, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: "Slide", owner: Type) -> str:
```

Comment:
```suggestion
    def __get__(self, instance: Any, owner: Type[object]) -> str:
```

Reply:
I used "Slide" as a forward declaration here. Isn't it correct approach? Could you please explain why "Any" is preferable here?